jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/IntermediateEventTest.java

```diff
@@ -1206,7 +1206,7 @@ public void testTimerBoundaryEventInterruptingOnTaskCancelTimer() throws Excepti
         ksession = restoreSession(ksession, true);
         ksession.getWorkItemManager().registerWorkItemHandler("Human Task", handler);
         timers = getTimerManager(ksession).getTimers();
-        assertEquals(0, timers.size());
+        assertTrue(timers == null || timers.isEmpty());
```

Comment:
I would rather avoid using assertTrue (if you are not asserting a value of boolean variable, of course). Reason is that when assertTrue fails, it does not tell you why - was is null or empty? Please take a look at assertJ library, is provides decent asserts for (not only) collections.